src/transformState.js

```diff
@@ -1,11 +1,27 @@
 'use strict';
 
-/**
- * @param {Object} state
- * @param {Object[]} actions
- */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    for (const key in actions[i]) {
+      if (actions[i][key] === 'addProperties') {
```

Comment:
according to the checklist: 2. [CODE STYLE]: Use switch statement if you have limited amount of conditions.